defaults/main/main.yml

```diff
@@ -27,12 +19,15 @@ nginx_type: opensource
 # nginx_version: "-20*"
 
 # Specify whether you want to maintain your version of NGINX, upgrade to the latest version, or remove NGINX.
-# Can be used with `nginx_version` to fine tune control on which version of NGINX is installed/used on each playbook execution.
-# Using 'present' will install the latest version (or 'nginx_version') of NGINX on a fresh install.
-# Using 'latest' will upgrade NGINX to the latest version (that matches your 'nginx_version') of NGINX on every playbook execution.
-# Using 'absent' will remove NGINX from your system.
-# Default is present.
-nginx_state: present
+# Can be used with `nginx_version` to fine tune con
+# Start NGINX service.
+# Default is true.
+nginx_start: truetrol on which version of NGINX is installed/used on each playbook execution.
```

Comment:
Did `truetrol` come from part of line 22?

Reply:
Yep, might be what was causing tests to fail.